engine/src/execution_graph/logic_controllers/BatchAggregationProcessing.h

```diff
@@ -131,13 +149,20 @@ class ComputeAggregateKernel : public kernel {
 private:
     std::vector<AggregateKind> aggregation_types;
     std::vector<int> group_column_indices;
+    std::vector<std::string> aggregation_input_expressions;
+    std::vector<std::string> aggregation_column_assigned_aliases;
```

Comment:
assigned_aliases? assigned by who for what? this name isn't clear to me perhaps it will be more clear if you ad a comment describing what this is for

Reply:
This is not new code, just moved. The asigned aliases are asigned by the user in the query. Think:
`select max(colA) as the_best_max from ....`
the_best_mas would be the assigned alias. And if none is assigned by the user, we define a column name based on the aggregation and column.